lib/rack/request.rb

```diff
@@ -780,4 +752,6 @@ def x_forwarded_proto_priority
   end
 end
 
+# :nocov:
 require_relative 'multipart' unless defined?(Rack::Multipart)
```

Comment:
Do we still need this `defined?`?

Reply:
In fact can we move this to the top of the file?

Reply:
I don't think so.  I believe that will introduce circular requires.

Reply:
Multipart no longer depends on request though?

Reply:
Ah, in that case, it should be safe to move this to the top and remove the `defined?`

Reply:
Yes, I think so.